brands/README.md

```diff
@@ -1,16 +1,18 @@
 # Brands
 
-After creating an account and having access to the dashboard, the next step is to create a test **Brand** so you can create locations and add them to a Program in order to start tracking transactions.
+After creating an account and having access to the dashboard, the next step is to create a test **Brand** (Merchant) so you can create locations and add them to a Program in order to start tracking transactions. 
 
-The **Brand** (Merchant) object is used to aggregate locations and keep the status of the Brand consent. One of the requirements, when creating a Brand and before adding locations, is to obtain Brand consent from an authorized personnel (Brand User) and to provide the Brand with access to view the transactional data that is being shared.
+>Once you create a **Brand**, it cannot be deleted.
 
-You can only create new locations on live environment from Brands that you have invited to create an account have access to their card linked transactions. On test environment, the consent status is automatically approved so you can continue testing.
+The **Brand** object is used to aggregate locations and keep the status of the Brand consent. 
```

Comment:
`the status of` -> `track of`. 

The consent and brand user parts are important. We should add something like:
```
To track real-time transactions at a Brand's locations, you need to obtain consent from authorised personnel (Brand User), and provide the Brand with access to view transactional data. You can do this when creating the Brand (see below).
```

Reply:
brand consent is covered later in the post.   should we have it more than once?

Reply:
I think it's best to keep this very clear.

Reply:
added your sentence